### PR TITLE
Update de.po

### PR DESCRIPTION
**Pull request by Chronophylos**

### CODE REVIEW

File: de.po [hunk 1]
```diff
@@ -948,73 +941,69 @@ msgid "Some mod ID resolving tasks failed."
 msgstr "Einige Mod-IDs konnten nicht aufgelöst werden."
 
 #: src/api/logic/modplatform/flame/UrlResolvingTask.cpp:22
-#, fuzzy
 msgctxt "Flame::UrlResolvingTask|"
 msgid "Resolving URL..."
-msgstr "Löse Mod-IDs auf..."
+msgstr "Löse URL..."
```
Comment (Janrupf) [inline de.po]: I don't think this makes much sense

File: de.po [hunk 2]
```diff
@@ -948,73 +941,69 @@ msgid "Some mod ID resolving tasks failed."
...
 msgid "Couldn't find the end of the needle in the haystack..."
-msgstr ""
+msgstr "Das Ende der Nadel konnte nicht im Heuhaufen gefunden werden..."
 
 #: src/api/logic/modplatform/flame/UrlResolvingTask.cpp:103
-#, fuzzy
 msgctxt "Flame::UrlResolvingTask|"
 msgid "Resolving failed."
-msgstr "Löse Mod-IDs auf..."
+msgstr "Auflösung fehlgeschlagen."
```
Comment (Janrupf) [inline de.po]: I would suggest `Auflösen der Mod-IDs fehlgeschlagen`

File: de.po [hunk 3]
```diff
@@ -948,73 +941,69 @@ msgid "Some mod ID resolving tasks failed."
...
 
 #: src/api/logic/modplatform/flame/UrlResolvingTask.cpp:108
 msgctxt "Flame::UrlResolvingTask|"
 msgid "Resolving failed: missing package root element."
-msgstr ""
+msgstr "Auflösung fehlgeschlagen: Fehlendes Paket-Rootelement."
```
Comment (Janrupf) [inline de.po]: `Auflösen`

File: de.po [hunk 4]
```diff
@@ -948,73 +941,69 @@ msgid "Some mod ID resolving tasks failed."
...
 
 #: src/api/logic/modplatform/flame/UrlResolvingTask.cpp:113
 msgctxt "Flame::UrlResolvingTask|"
 msgid "Resolving failed: missing project element."
-msgstr ""
+msgstr "Auflösung fehlgeschlagen: Fehlendes Projektelement."
```
Comment (Janrupf) [inline de.po]: `Auflösen`

File: de.po [hunk 5]
```diff
@@ -948,73 +941,69 @@ msgid "Some mod ID resolving tasks failed."
...
 
 #: src/api/logic/modplatform/flame/UrlResolvingTask.cpp:120
 msgctxt "Flame::UrlResolvingTask|"
 msgid "Resolving failed: missing id attribute."
-msgstr ""
+msgstr "Auflösung fehlgeschlagen: Fehlendes ID-Attribut."
```
Comment (Janrupf) [inline de.po]: `Auflösen`

File: de.po [hunk 6]
```diff
@@ -948,73 +941,69 @@ msgid "Some mod ID resolving tasks failed."
...
 
 #: src/api/logic/modplatform/flame/UrlResolvingTask.cpp:125
 msgctxt "Flame::UrlResolvingTask|"
 msgid "Resolving failed: missing file attribute."
-msgstr ""
+msgstr "Auflösung fehlgeschlagen: Fehlendes Datei-Attribut."
```
Comment (Janrupf) [inline de.po]: `Auflösen`

File: de.po [hunk 7]
```diff
@@ -4911,17 +4868,15 @@ msgstr "Minecraft.jar Mods (*.zip *.jar)"
 
 #: src/application/pages/instance/VersionPage.cpp:359
 #: src/application/pages/instance/VersionPage.cpp:444
-#, fuzzy
 msgctxt "VersionPage|"
 msgid "No Fabric Loader versions are currently available."
-msgstr "Keine Versionen sind momentan verfügbar."
+msgstr "Keine Fabric-Loader Versionen sind momentan verfügbar."
```
Comment (Janrupf) [inline de.po]: `Es sind momentan keine Fabric-Loader versionen verfügbar` - sounds better in my opinion

File: de.po [hunk 8]
```diff
@@ -217,21 +217,20 @@ msgid "Accounts"
...
-"hinzuzufügen.</p></body></html>"
+"Willkommen!\n"
+"Solltest du neu sein, kannst du die \"Hinzufügen\"-Schaltfläche drücken um "
+"dein Mojang- oder Minecraft-Konto hinzuzufügen."
 
 #: src/application/pages/global/AccountListPage.cpp:77
 msgctxt "AccountListPage|"
 msgid "Context menu"
-msgstr ""
+msgstr "Kontext Menü"
```
Comment (DAmNRelentless) [inline de.po]: If you write "Hauptfenster" and "Symbolleiste", this should be "Kontextmenü"

File: de.po [hunk 9]
```diff
@@ -948,73 +941,69 @@ msgid "Some mod ID resolving tasks failed."
...
 msgctxt "Flame::UrlResolvingTask|"
 msgid "Failed to resove projectId as a number."
-msgstr ""
+msgstr "Die ProjektID konnte nicht als Nummer aufgelöst werden."
 
 #: src/api/logic/modplatform/flame/UrlResolvingTask.cpp:139
 msgctxt "Flame::UrlResolvingTask|"
 msgid "Failed to resove fileId as a number."
-msgstr ""
+msgstr "Die FeldID konnte nicht als Nummer aufgelöst werden."
```
Comment (DAmNRelentless) [inline de.po]: "DateiID" since it's "fileID"

File: de.po [hunk 10]
```diff
@@ -1474,18 +1462,17 @@ msgstr ""
...
 msgctxt "InstanceSettingsPage|"
 msgid "Open Global Settings"
-msgstr "Konsoleneinstellungen"
+msgstr "Öffne Globale Einstellungen"
 
 #: src/application/pages/instance/InstanceSettingsPage.ui:32
 msgctxt "InstanceSettingsPage|"
 msgid "The settings here are overrides for global settings."
-msgstr ""
+msgstr "Die Einstellungen hier überschreiben die globalen Einstellungen."
```
Comment (DAmNRelentless) [inline de.po]: "Diese Einstellungen überschreiben die globalen Einstellungen." sounds better imo. The english one is a bit weird tho.

File: de.po [hunk 11]
```diff
@@ -1802,7 +1789,7 @@ msgstr "Du siehst deine Sprache nicht oder die Qualität ist schlecht?"
 #: src/application/widgets/LanguageSelectionWidget.cpp:53
 msgctxt "LanguageSelectionWidget|"
 msgid "Help us with translations!"
-msgstr ""
+msgstr "Hilf uns mit Übersetzungen!"
```
Comment (DAmNRelentless) [inline de.po]: "Hilf uns beim Übersetzen!"

File: de.po [hunk 12]
```diff
@@ -2742,133 +2729,115 @@ msgid "Aborted by user."
...
 msgctxt "ModFolderPage|"
 msgid "&Add"
 msgstr "&Hinzufügen"
 
 #: src/application/pages/instance/ModFolderPage.ui:102
-#, fuzzy
 msgctxt "ModFolderPage|"
 msgid "Add mods"
-msgstr "Loader-Mods"
+msgstr "Füge Mods hinzu"
```
Comment (DAmNRelentless) [inline de.po]: "Mods hinzufügen"

File: de.po [hunk 13]
```diff
@@ -2742,133 +2729,115 @@ msgid "Aborted by user."
...
 msgctxt "ModFolderPage|"
 msgid "&Remove"
 msgstr "&Entfernen"
 
 #: src/application/pages/instance/ModFolderPage.ui:110
-#, fuzzy
 msgctxt "ModFolderPage|"
 msgid "Remove selected mods"
-msgstr "Entferne ausgewähltes Paket"
+msgstr "Entferne ausgewählte Mods"
```
Comment (DAmNRelentless) [inline de.po]: "Ausgewählte Mods entfernen"

File: de.po [hunk 14]
```diff
@@ -2742,133 +2729,115 @@ msgid "Aborted by user."
...
 msgctxt "ModFolderPage|"
 msgid "&Enable"
-msgstr "Einschalten"
+msgstr "Aktivier&en"
 
 #: src/application/pages/instance/ModFolderPage.ui:118
 msgctxt "ModFolderPage|"
 msgid "Enable selected mods"
-msgstr ""
+msgstr "Aktiviere ausgwählte Mods"
```
Comment (DAmNRelentless) [inline de.po]: "Ausgewählte Mods aktivieren"

File: de.po [hunk 15]
```diff
@@ -2742,133 +2729,115 @@ msgid "Aborted by user."
...
 msgid "&Disable"
-msgstr "Ausschalten"
+msgstr "&Deaktivieren"
 
 #: src/application/pages/instance/ModFolderPage.ui:126
-#, fuzzy
 msgctxt "ModFolderPage|"
 msgid "Disable selected mods"
-msgstr "Keine Instanz ausgewählt"
+msgstr "Deaktiviere ausgewählte Mods"
```
Comment (DAmNRelentless) [inline de.po]: "Ausgewählte Mods deaktivieren"

File: de.po [hunk 16]
```diff
@@ -2742,133 +2729,115 @@ msgid "Aborted by user."
...
 
 #: src/application/pages/instance/ModFolderPage.ui:131
-#, fuzzy
 msgctxt "ModFolderPage|"
 msgid "View &Configs"
-msgstr "Einstellungen öffnen"
+msgstr "Öffne Einstellungen"
```
Comment (DAmNRelentless) [inline de.po]: "Einstellungen öffnen" was better, it's better grammar, same like the other ones.

File: de.po [hunk 17]
```diff
@@ -2742,133 +2729,115 @@ msgid "Aborted by user."
...
 
 #: src/application/pages/instance/ModFolderPage.ui:139
-#, fuzzy
 msgctxt "ModFolderPage|"
 msgid "View &Folder"
-msgstr "&Ordner öffnen"
+msgstr "Öffne Ordner"
```
Comment (DAmNRelentless) [inline de.po]: "Ordner öffnen" was right

File: de.po [hunk 18]
```diff
@@ -2878,15 +2847,15 @@ msgstr "Filter:"
 #: src/application/pages/instance/ModFolderPage.cpp:168
 msgctxt "ModFolderPage|"
 msgid "Context menu"
-msgstr ""
+msgstr "Kontext Menü"
```
Comment (DAmNRelentless) [inline de.po]: "Kontextmenü"

File: de.po [hunk 19]
```diff
@@ -4168,7 +4137,7 @@ msgstr "Bildschirmfotos"
 #: src/application/pages/instance/ScreenshotsPage.cpp:273
 msgctxt "ScreenshotsPage|"
 msgid "Context menu"
-msgstr ""
+msgstr "Kontext Menü"
```
Comment (DAmNRelentless) [inline de.po]: "Kontextmenü"

File: de.po [hunk 20]
```diff
@@ -4359,7 +4320,7 @@ msgstr "Server"
 #: src/application/pages/instance/ServersPage.cpp:604
 msgctxt "ServersPage|"
 msgid "Context menu"
-msgstr ""
+msgstr "Kontext Menü"
```
Comment (DAmNRelentless) [inline de.po]: "Kontextmenü"

File: de.po [hunk 21]
```diff
@@ -4516,22 +4477,19 @@ msgstr ""
...
 msgid "Twitch URL:"
-msgstr "Twitch"
+msgstr "Twitch URL:"
 
 #: src/application/pages/modplatform/TwitchPage.ui:48
-#, fuzzy
 msgctxt "TwitchPage|"
 msgid "Check"
-msgstr "Überprüfen"
+msgstr "Überprüfe"
```
Comment (DAmNRelentless) [inline de.po]: "Überprüfen" was right

File: de.po [hunk 22]
```diff
@@ -4875,7 +4832,7 @@ msgstr "Instanz-Profil konnte nicht geladen werden."
 #: src/application/pages/instance/VersionPage.cpp:146
 msgctxt "VersionPage|"
 msgid "Context menu"
-msgstr ""
+msgstr "Kontext Menü"
```
Comment (DAmNRelentless) [inline de.po]: "Kontextmenü"

File: de.po [hunk 23]
```diff
@@ -4911,17 +4868,15 @@ msgstr "Minecraft.jar Mods (*.zip *.jar)"
 
 #: src/application/pages/instance/VersionPage.cpp:359
 #: src/application/pages/instance/VersionPage.cpp:444
-#, fuzzy
 msgctxt "VersionPage|"
 msgid "No Fabric Loader versions are currently available."
-msgstr "Keine Versionen sind momentan verfügbar."
+msgstr "Es sind momentan keine Fabric-Loader versionen verfügbar."
```
Comment (DAmNRelentless) [inline de.po]: "Es sind momentan keine Fabric-Versionen verfügbar."
If you want to keep the loader, make "Versionen" with a capital 'V'

File: de.po [hunk 24]
```diff
@@ -5230,7 +5182,7 @@ msgstr "Kein MCEdit gefunden oder aufgesetzt!"
 #: src/application/pages/instance/WorldListPage.cpp:76
 msgctxt "WorldListPage|"
 msgid "Context menu"
-msgstr ""
+msgstr "Kontext Menü"
```
Comment (DAmNRelentless) [inline de.po]: "Kontextmenü"

**Comment by Chronophylos** [comment]
It seems like `&` makes chars to be underlined. Does that also make them to keybinds?

**Comment by Janrupf** [comment]
I think so

**Pull request merged**